openbb_platform/providers/yfinance/openbb_yfinance/models/crypto_historical.py

```diff
@@ -72,12 +72,21 @@ def extract_data(
         **kwargs: Any,
     ) -> List[Dict]:
         """Return the raw data from the Yahoo Finance endpoint."""
-        if "-" not in query.symbol:
-            position = len(query.symbol) - 3
-            query.symbol = query.symbol[:position] + "-" + query.symbol[position:]
+
+        tickers = query.symbol.split(",") if "," in query.symbol else [query.symbol]
+        new_tickers = []
+        for ticker in tickers:
+            if "-" not in ticker:
+                position = len(ticker) - 3
+                new_ticker = ticker[:position] + "-" + ticker[position:]
```

Comment:
You can also just combine this as:

new_ticker = ticker[:-3] + "-" + ticker[-3:]

The negative slicing gets rid of the need for the intermediate calc.